challenges/objects-arrays.js

```diff
@@ -83,43 +92,43 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
-/* Request 1: .forEach()
-
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
+/* Request 1: .forEach()
+The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and 
+scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 */
 const animalNames = [];
+zooAnimals.forEach((element) => animalNames.push(`Name: ${element.animal_name}, Scientific: ${element.scientific_name}`));
 console.log(animalNames);
 
-/* Request 2: .map()    
-
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
+/* Request 2: .map()    
+The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and 
+map over each name to convert them all to lower case.  Log the resut.
 */
-
-const lowerCase = [];
+const lowerCase = zooAnimals.map((element) => element.animal_name.toLowerCase());
 console.log(lowerCase); 
 
-/* Request 3: .filter() 
-
-The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
+/* Request 3: .filter() 
+The zoos are concerned about animals with a lower population count. Find out which animals have a population less than 5.
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter((element) => element.population < 5);
 console.log(largerPopulation);
 
-/* Request 4: .reduce() 
 
+/* Request 4: .reduce() 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
-
 */
-const populationTotal = 0;
-console.log(populationTotal);
+const reducer = (accumulator, currentValue) => accumulator + currentValue;  // Handy-dandy MDN reduce helper function
+const populationTotal = []; // Empty array for easy adding and reducing later on
+zooAnimals.forEach((element) => populationTotal.push(element.population)); // Loop through root array using forEach, pushing population values into new array
```

Comment:
In the future I would like  to see you just use the .reduce method with an anonymous function. Much cleaner and shorter code.